File: python/1115.py
import sys
while True:
    x, y = list(map(int, input().split(' ')))
    if x>0:
        if y>0:
            print("primeiro")
        else:
            print("quarto")
        
    elif x<0:
        if y>0:
            print("segundo")
        else:
            print("terceiro")
        
    else:
        sys.exit()
    

File: c/1115.c
#include <stdio.h>

int main(){
    while (1){
        int x, y;
        scanf("%d %d", &x, &y);
        if (x>0){
            if (y>0){
                printf("primeiro\n");
            }else{
                printf("quarto\n");
            }
        }else if(x<0){
            if (y>0){
                printf("segundo\n");
            }else{
                printf("terceiro\n");
            }
        }else{
            break;
        }
    }
}